### ISSUE TITLE
Fix flaky base-carousel e2e test

### ISSUE THREAD

**Issue by jridgewell** [ampproject/amphtml#22845]
https://github.com/ampproject/amphtml/pull/22809 skips a flaky BaseCarousel e2e test. I can't figure out why the changes would cause any failures, the carousel itself uses native APIs to get the bounding client rect.

**Comment by nainar** [comment]
Dupe of https://github.com/ampproject/amphtml/issues/24195
